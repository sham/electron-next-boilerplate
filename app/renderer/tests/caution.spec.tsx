import * as React from 'react'
import * as Renderer from 'react-test-renderer'

import Hello from '../pages/hello'

test('page Caution', () => {
  const component = Renderer.create(
    <Hello />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
