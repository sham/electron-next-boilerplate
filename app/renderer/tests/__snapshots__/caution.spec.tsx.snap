// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page Caution 1`] = `
Array [
  <h1>
    Hello!
  </h1>,
  <p>
    You're welcome.
  </p>,
  <a
    href="/start"
    onClick={[Function]}
  >
    Back
  </a>,
]
`;
